### ISSUE TITLE
Pohatu jump glitch

### ISSUE THREAD

**Issue by ShadowOfNight** [TheLegendOfMataNui/game-issues#19]
Pohatu performs a "skip" after scaling a ledge or jumping, resulting in the landing/jumping sound being played twice in quick succession.  

NOTE: I tried using Onua and Gali to see if this glitch was level related, but it only occurred when using Pohatu.

**Comment by ShadowOfNight** [comment]
This is still a problem in the beta, and it's more pronounced.  The landing sound plays 3 times, and if you keep pressing the forward key during the scale animation, Pohatu will perform a roll, as though he was landing from a running jump.

**Comment by LennyTheBionicleLover** [comment]
It’s been three years. Is this fixed yet?

**Comment by ShadowOfNight** [comment]
No, this has not been fixed.  If an issue is open, it has not been fixed yet.  This issue has not been looked into and may very well be addressed in a following patch.  For now, it is not currently being looked into as it is outside the scope of our current patch.
